portfolio/src/main/webapp/index.html

```diff
@@ -13,7 +13,7 @@
     <link rel="stylesheet" href="style.css" />
     <script src="script.js"></script>
   </head>
-  <body onload="getComments()">
+  <body onload="getComments(); checkLoginStatus()">
```

Comment:
Preferably move the `onload` functions to the comments-container below - make it more context specific (since it's not related to the body)

Reply:
I don't think they get called if I move them to the comments-container because onload doesn't work with div.